src/config.rs

```diff
@@ -153,6 +247,11 @@ warning-policy = "error"
             traverse_parent_directories: true,
             exclude: vec![Regex::new(r"google\.com").unwrap()],
             user_agent: String::from("Internet Explorer"),
+            http_headers: vec![(
```

Comment:
Thanks for adding a test to make sure we can parser the configuration :+1:

Reply:
I'm looking forward to add a separate test for that. Probably, we should try to resolve some `cartes.io` link, which won't work without `Accept: text/html`. I'll get to it tomorrow, it's kind o late in my place :)